sys/exec.c

```diff
@@ -242,6 +230,19 @@ int do_exec(const exec_args_t *args) {
     proc_populate(proc, td);
   }
 
+  /* Prepare file descriptor table */
+  /* TODO: Copy/share file descriptor table! */
```

Comment:
I find this comment unclear. Unix-like `exec` does not modify descriptor table unless a descriptor was marked with `O_CLOEXEC` flag on `open`. I understand that our `exec` has unusual use case – i.e. it's used to start user-space programs from kernel, therefore we need to allocate fresh descriptor table.